akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -16,21 +17,24 @@ import scala.concurrent.duration.FiniteDuration
 import scala.language.higherKinds
 import scala.language.implicitConversions
 import akka.stream.stage.Stage
+import akka.stream.impl.StreamLayout
 
 /** Java API */
 object Source {
 
   import scaladsl.JavaConverters._
 
+  val factory: SourceCreate = new SourceCreate {}
```

Comment:
I encountered trouble with missing static forwarders for Java when just mixing in the traits, so I decided to go the safe route, which means offering the methods on an instance.